tests/unit_tests/datasource/dao_tests.py

```diff
@@ -106,7 +106,6 @@ def test_get_datasource_sqlatable(session_with_data: Session) -> None:
     result = DatasourceDAO.get_datasource(
         datasource_type=DatasourceType.TABLE,
         datasource_id=1,
-        session=session_with_data,
```

Comment:
The `db.session` is mocked [here](https://github.com/apache/superset/blob/master/tests/unit_tests/conftest.py#L65) and is included as a fixture to the test.